### PR TITLE
cm / Adding OAuth to Ada Books

### PR DESCRIPTION
**Pull request by CheezItMan**
I went through the exercise and added OAuth to Ada Books for testing!

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -5,17 +5,40 @@ def login_form
 
   def login
     username = params[:user][:username]
+
     user = User.find_by(username: username)
-    user = User.create(username: username) if user.nil?
+    if user.nil?
+      flash_msg = "Welcome new user"
```
Comment (CheezItMan) [inline app/controllers/users_controller.rb:11]: You should change this flash notice to `"welcome #{user.username}"

**Review by CheezItMan — COMMENTED** [review]
Maybe change the flash notices!

**Review by tildeee — CHANGES_REQUESTED** [review]
Hm, seems pretty good, but it needs more pictures of Samson

**Comment by CheezItMan** [comment]
> Hm, seems pretty good, but it needs more pictures of Samson

Agreed, but I don't have any, as my partner has neglected their Samson duties.

**Pull request merged**